Hunting Queries/MultipleDataSources/NetworkConnectiontoOMIPorts.yaml

```diff
@@ -51,5 +54,12 @@ query: |
   | parse VM_s with * '/' VM 
   | project TimeGenerated, SourceIp = SrcIP_s, DestinationIp = DestIP_s, DestinationPort = DestPort_d, Protocol = L7Protocol_s, NSGRule_s, VM, Type
   | extend Timestamp = TimeGenerated, HostCustomEntity = VM, IPCustomEntity = SourceIp
+  ),
+  AzureDiagnostics
+  | where Category == "AzureFirewallNetworkRule" and OperationName == "AzureFirewallNatRuleLog"
+  | parse msg_s with Protocol ' request from ' SourceIp ':' SourcePort ' to ' DestinationIp ':' DestinationPort " was " Action " to " InternalIP ':' InternalPort
+  | where DestinationPort in (Port)
+  | project TimeGenerated, SourceIP, DestinationIP, DestinationPort, Protocol, Action, Resource
```

Comment:
We probably want to correct the case of the Source and Destination IP to be the same as what we have in the parse statement.
I guess we probably need SourceIp and DestinationIp in the project statement. Thanks.